modules/file-system/filestore/README.md

```diff
@@ -26,6 +26,21 @@ creation will fail when you run `terraform apply`.
 
 [tiers]: https://cloud.google.com/filestore/docs/service-tiers
 
+### Filestore mount options
+Toolkit uses [default mount options](https://linux.die.net/man/8/mount) for all tiers
+services. Filestore recommends service specific mount options for better performance.
```

Comment:
s/for all tiers services/for all firestore tiers/

Reply:
I reworded the documentation. I am not sure if we should put details like what are the recommended options for which tiers. We have a pointer to the recommended options from filestore. Here I only specified how customers can pass this parameter to update the default mount options. If they want to use different file system they can refer the filestore documentation and pass the tier specific recommended option in similar way.

LMKWYT.

Reply:
I would still replace "all tier services" for "all filestore tiers"

Reply:
Done. Updated "all filestore tiers" to "all tier services"